test/cpp/end2end/xds/xds_core_end2end_test.cc

```diff
@@ -1424,6 +1426,10 @@ TEST_P(XdsFederationLoadReportingTest, FederationMultipleLoadReportingTest) {
                        kNewListenerTemplate);
   InitClient(builder);
   CreateAndStartBackends(2, /*xds_enabled=*/true);
+  ASSERT_TRUE(backends_[0]->notifier()->WaitOnServingStatusChange(
```

Comment:
I think we should handle this case by using wait_for_ready as well.

Reply:
done